plugins/embed-optimizer/hooks.php

```diff
@@ -147,29 +147,6 @@ function embed_optimizer_lazy_load_scripts(): void {
 	wp_print_inline_script_tag( $js, array( 'type' => 'module' ) );
 }
 
-/**
- * Generates a user-level error/warning/notice/deprecation message.
- *
- * Generates the message when `WP_DEBUG` is true.
- *
- * @since 0.1.0
- *
- * @param string $function_name The function that triggered the error.
- * @param string $message       The message explaining the error.
- *                              The message can contain allowed HTML 'a' (with href), 'code',
- *                              'br', 'em', and 'strong' tags and http or https protocols.
- *                              If it contains other HTML tags or protocols, the message should be escaped
- *                              before passing to this function to avoid being stripped {@see wp_kses()}.
- * @param int    $error_level   Optional. The designated error type for this error.
- *                              Only works with E_USER family of constants. Default E_USER_NOTICE.
- */
-function embed_optimizer_trigger_error( string $function_name, string $message, int $error_level = E_USER_NOTICE ): void {
-	if ( ! function_exists( 'wp_trigger_error' ) ) {
-		return;
-	}
-	wp_trigger_error( $function_name, $message, $error_level );
-}
-
```

Comment:
This is actually WP 6.3-compat code, as `wp_trigger_error()` was introduced in 6.4.